test/e2e/cluster_config.sh

```diff
@@ -47,19 +47,37 @@ clusters:
     karpenter_pools_enabled: "true"
     okta_auth_client_id: "kubernetes.cluster.teapot-e2e"
     teapot_admission_controller_validate_pod_images_soft_fail_namespaces: "^kube-system$"
+    eks_okta_identity_provider: "false" # disabled to speed up EKS cluster creation for e2e.
   criticality_level: 1
   environment: e2e
   id: ${CLUSTER_ID}
   infrastructure_account: ${INFRASTRUCTURE_ACCOUNT}
   lifecycle_status: ${2}
   local_id: ${LOCAL_ID}
   node_pools:
-  - discount_strategy: none
+  $(if [ "${CLUSTER_PROVIDER}" == "zalando-eks" ]; then
+cat <<EOFF
+- config_items:
+      labels: dedicated=cluster-seed
+      taints: dedicated=cluster-seed:NoSchedule
+    discount_strategy: none
+    instance_types:
+    - "m6i.xlarge"
+    max_size: 99
+    min_size: 2
```

Comment:
will CLM not complain here, keeping min/max size not a multiple of 3? (it usually does when creating pools)

Reply:
` profile: worker-combined` does not have this restriction because it has an ASG spanning all zones as opposed to one ASG per zone.

It's actually not intended to autoscale as what fits on the seed node should not. It's kind of like a master node in non-eks setup as concept.

Reply:
Ah, got it 🙂